package.json

```diff
@@ -19,11 +19,16 @@
     "zooniverse"
   ],
   "scripts": {
-    "start": "NODE_ENV=development check-engines && check-dependencies && webpack-dashboard -p 3777 -- babel-node server.js",
-    "test": "NODE_ENV=development BABEL_ENV=test mocha",
+    "start": "export NODE_ENV=development; check-engines && check-dependencies && webpack-dashboard -- babel-node ./server.js",
+    "_build": "HEAD_COMMIT=$(git rev-parse --short HEAD); check-engines && check-dependencies && rimraf dist; webpack --config ./webpack.production.config.js -p",
```

Comment:
`rimraf` needed if using CleanWebpackPlugin in [webpack.production.config](https://github.com/zooniverse/pfe-lab/blob/master/webpack.production.config.js#L21)? I think duplicative, can omit rimraf here in script.